pkg/dom0-ztools/rootfs/bin/mdev-zvol.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+if [ "$ACTION" = "add" ]; then
+  datasetName=$(/lib/udev/zvol_id "/dev/$MDEV")
+  retVal=$?
+  if [ $retVal -ne 0 ]; then
+    echo "$datasetName"
```

Comment:
Is this so an "add" can be used as a "get"? Might make sense to add a "get" action to make that more explicit.

If you also need an action which does a "addorget" which ensures that it exists by adding it only if need be it would make sense to make that an explicit action with a different name than "add".

Reply:
Log may be obtained if we will create `/dev/mdev.log` file. It is not `get` action, just line in output.